packages/modules/vehicles/ovms/soc.py

```diff
@@ -10,14 +10,32 @@
 from modules.common.configurable_vehicle import ConfigurableVehicle
 from modules.vehicles.ovms import api
 from modules.vehicles.ovms.config import OVMS, OVMSConfiguration
+import datetime
+from json import dumps
 
-
+date_fmt = '%Y-%m-%d %H:%M:%S'
 log = logging.getLogger(__name__)
 
 
 def fetch(vehicle_update_data: VehicleUpdateData, config: OVMS, vehicle: int) -> CarState:
-    soc, range, soc_ts = api.fetch_soc(config, vehicle)
-    log.info("Result: soc/range=" + str(soc)+"/" + str(range) + "@" + soc_ts)
+    soc, range, soc_ts, kms, vehicle12v = api.fetch_soc(config, vehicle)
+    logs = {}
+    logs['timestamp'] = datetime.datetime.now().strftime(date_fmt)
+    logs['vehicle'] = str(vehicle)
+    logs['module_name'] = config.name
+    logs['module_type'] = config.type
+    logs['soc'] = str(soc)
+    logs['range'] = str(range)
+    logs['km_stand'] = str(kms)
+    logs['car_timestamp'] = soc_ts
+    logs['SoC12V'] = str(vehicle12v)
+    log.info("##LOG" + str(vehicle) + ":" + dumps(logs))
+
+    log.info("\nResult: soc=" + str(soc) +
+             ", range=" + str(range) + "km" +
+             ", car_timestamp=" + soc_ts +
+             ", km-Stand=" + str(kms) + "km" +
+             ", 12V-Batt=" + str(vehicle12v) + "V")
```

Comment:
```suggestion
```
Bitte nicht mehrfach loggen. vehicle12v ist 4-mal in einem Logzyklus enthalten: hier und in api.py in Z.173, im Log der Antwort in Z. 124 und im Session-Log der Abfrage in Z. 113 (das sollte als einzige Log-Meldung ausreichen, denn dann wird auch der Modul-Code nicht durch unzählige Logmeldungen aufgebläht.)